client/app/components/ParameterMappingInput.jsx

```diff
@@ -301,15 +316,14 @@ class EditMapping extends React.Component {
     }
 
     this.setState({ mapping, inputError });
-  }
+  };
 
-  get content() {
+  getContent() {
```

Comment:
Why?

Reply:
Because when it's property getter - it's less clear to understand that it renders something. Probably I even had to call it `renderContent` (as I did below :thinking: )